docs/docs-content/clusters/pcg/manage-pcg/pcg-upgrade.md

```diff
@@ -0,0 +1,82 @@
+---
+sidebar_label: "Upgrade a PCG"
+title: "Upgrade a PCG"
+description: "Learn how to upgrade a Private Cloud Gateway (PCG) in Palette."
+hide_table_of_contents: false
+sidebar_position: 40
+tags: ["pcg"]
+---
+
+A PCG upgrade event can be broken down into two categories: Palette agent updates and cluster profile updates. Depending
+on the upgrade event, different steps are required to complete the upgrade.
+
+Palette agent updates are tied to the Palette platform version. Self-hosted Palette instances must upgrade to the next
+minor or major version to receive the latest Palette agent and cluster profile updates that are compatible with the new
+version of Palette. For example, if you are on Palette 4.1.12, you must upgrade to next minor version, Palette 4.2.0 or
+greater to receive the latest Palette agent updates that are compatible with Palette 4.2.x. We recommend you check for
+cluster profile updates after upgrading your self-hosted Palette instance to a new version.
+
+The table below outlines a high-level overview of the upgrade process for a PCG and each of its components.
+
+| Component       | User Action Required? | Expected Downtime? | Description                                                                                                                                                                                                                                                                                                    |
+| --------------- | --------------------- | ------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Palette Agent   | No                    | No                 | Automatically updated unless platform updates or cluster updates are paused. Refer to the [Pause Platform Upgrades](../../cluster-management/platform-settings/pause-platform-upgrades.md) to learn more about pausing updates.                                                                                |
+| Cluster Profile | Yes                   | Potentially        | Manually approve the cluster profile update to apply the latest cluster profile changes. Refer to the [Pause Platform Upgrades](../../cluster-management/platform-settings/pause-platform-upgrades.md) to learn more about pausing updates. Not all Palette version updates introduce cluster profile changes. |
```

Comment:
"Potential Expected Downtime?" I'm not aware of potential downtime. New machines are supposed to be provisioned, and the PCG components will be moved there.

Reply:
I updated the heading to state PCG Downtime as the context is only for the PCG. Great call out 👍🏻